Android/ConstraintLayout/source/ConstraintLayout/ConstraintLayout.csproj

```diff
@@ -44,6 +51,45 @@
     <Reference Include="System.Xml" />
     <Reference Include="System.Core" />
     <Reference Include="Mono.Android" />
+    <Reference Include="Xamarin.Android.Support.Annotations">

```

Comment:
All these References with HintPath's need to get deleted.  The PackageReference will implicitly add these.